src/main/resources/application.yml

```diff
@@ -84,3 +84,9 @@ management:
 sqs:
   provider: aws
   endpoint.region: eu-west-2
+
+video-link-booking:
+  day-start: "09:00"
+  day-end: "16:00"
+  step: "PT15M"
+  max-alternatives: 3
```

Comment:
I like the use of config, but maybe worth putting these options in a sub-group that indicates they are only used when checking availability or finding options? As it stands, it suggests to me that all VLBs are between these times?!

Reply:
yep.  Will do.